addons/event/models/event_event.py

```diff
@@ -632,6 +634,12 @@ def _check_closing_date(self):
             if event.date_end < event.date_begin:
                 raise ValidationError(_('The closing date cannot be earlier than the beginning date.'))
 
+    @api.constrains('event_url')
+    def _check_event_url(self):
+        for line in self:
+            if line.event_url and not re.match(tools.mail.TEXT_URL_REGEX, line.event_url):
+                raise ValidationError(_('The Event URL is not valid.'))
+
```

Comment:
Ah yes, what about the https in the end? will it be set automatically?

Reply:
I was wondering if we could do something like in whatsapp to 'force' the https when missing? (instead of the constraint?)

Check `_onchange_website_url`

What do you think?